### PR TITLE
Bavarian

### PR DESCRIPTION
**Pull request by hanzlan**
added coordinates and countries

### CODE REVIEW

File: languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/uppe1398/md.ini [hunk 1]
```diff
@@ -8,3 +8,5 @@ macroareas =
 	Eurasia
 countries = 
 	Austria (AT)
+links =
+	[German (Upper Austrian)](https://wals.info/languoid/lect/wals_code_gau)
```
Comment (xrotwang) [inline languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/uppe1398/md.ini:12]: Good find! I wasn't even aware we had this in WALS. The liks to WALS are not really curated here, though, but [harvested from WALS](https://github.com/glottolog/pyglottolog/blob/15e46bf283ba1100b46ae8f940f8e75cd85812d7/src/pyglottolog/links/util.py#L105-L111). So to make this change permanent, you'd have to file an issue at https://github.com/cldf-datasets/wals/issues

File: languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini [hunk 2]
```diff
@@ -9,6 +9,6 @@ macroareas =
 countries = 
 	Austria (AT)
 links = 
+	[German (Viennese)](https://wals.info/languoid/lect/wals_code_gvi)
```
Comment (xrotwang) [inline languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini:12]: See above.

Comment (hanzlan) [reply languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini:12]: The division of Central Bavarian into Upper Austrian and Viennese is a bit random though. The main division of Central Bavarian dialects is into Western Central and Eastern Central Bavarian with the boundary running north-south between Passau and Linz. (Source: https://commons.wikimedia.org/wiki/File:Bairischer_Sprachraum2.PNG). Then we could place the city dialect Viennese into Eastern Central Bavarian, but it's a bit hard to assign Upper Austrian, as this is situated right at the boundary between the Western and Eastern dialects.

So we could do something like that:

Bavarian
- Central Bavarian
-- Western Central Bavarian
--- Munich German
-- Eastern Central Bavarian
--- Viennese German

Comment (xrotwang) [reply languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini:12]: The dialect inventories in Glottolog are often not really complete or systematic. Often these inventories are driven by "need", i.e. by what varieties linguists write about or do research with. So, the fact that Upper Austrian and Viennese are included in WALS is good reason to assign glottocodes to these varieties - which means including them as "dialects".

Regarding the Western/Eastern split: I'd leave this to @d97hah to decide. Renaming "Viennese German" makes sense, I think.

Comment (hanzlan) [reply languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/vien1238/md.ini:12]: On the other hand it is simply common to use the term "Viennese German" disregarding the linguistic affiliation.

**Comment by d97hah** [comment]
Ok following Wiesinger 1990:460-482 I propose the following (I hope the
format is self-explanatory, if not let me know). If you need further
divisions in e.g. W C Bavarian, like Munich, let me know

Bavarian [bava1246] bar
    Central Bavarian [cent1967]
    North Bavarian [nort2634]
    South Bavarian [sout2632]
Bavarian [bava1246] bar; Central Bavarian [cent1967]
    West Central Bavarian []
    East Central Bavarian []
Bavarian [bava1246] bar; Central Bavarian [cent1967]; East Central Bavarian
[]
    Lower Austria-Vienna Bavarian [vien1238]
    Upper Austria Bavarian [uppe1398]
Bavarian [bava1246] bar; Central Bavarian [cent1967]; East Central Bavarian
[]; Upper Austria Bavarian [uppe1398]
    Transylvanian Landler [land1263]
    Hausruckviertel []
Bavarian [bava1246] bar; South Bavarian [sout2632]
    Carinthian Bavarian []
    Tyrol Bavarian []
Bavarian [bava1246] bar; South Bavarian [sout2632]; Tyrol Bavarian []
    Gottscheerisch [gott1234]
    Austrian-Italian Tyrol Bavarian []


Pada tanggal Kam, 26 Nov 2020 pukul 12.25 Andreas Hanzl <
notifications@github.com> menulis:

**Comment by hanzlan** [comment]
This sounds very good! The only remark I would add here is the division of South Bavarian. [Moric](https://www.nomos-elibrary.de/10.24989/0014-2492-2011-12-21/the-maintenance-of-gottscheerisch-the-dialect-of-gottschee-germans-in-slovenia-austria-germany-canada-and-the-usa-jahrgang-68-2011-heft-1-2) states that the settlers of Gottschee came from the region around Upper Carinthia/East Tyrol starting from the 1430s, which is the transition zone between Carinthian and Tyrolean. For the last 600 years Gottscheerisch evolved rather independently. So I wouldn't directly classify Gottscheerisch under Tyrol Bavarian, but rather keep these three dialects as separate forms and don't implement the dialect Austrian-Italian Tyrol Bavarian [].

Bavarian [bava1246] bar; South Bavarian [sout2632]
- Carinthian Bavarian []
- Gottscheerisch [gott1234]
- Tyrol Bavarian []

**Comment by d97hah** [comment]
Yes that's a better solution, I'll PR in a sec

Pada tanggal Kam, 26 Nov 2020 pukul 14.49 Andreas Hanzl <
notifications@github.com> menulis:

**Comment by xrotwang** [comment]
@d97hah do you think we should add the [Moric](https://www.nomos-elibrary.de/10.24989/0014-2492-2011-12-21/the-maintenance-of-gottscheerisch-the-dialect-of-gottschee-germans-in-slovenia-austria-germany-canada-and-the-usa-volume-68-2011-issue-1-2) reference? Might be useful as source for endangerment status as well as classification.

**Comment by d97hah** [comment]
Yes already done though not committed in a PR yet, will be the next time
hh.bib is in a PR :-)

Pada tanggal Kam, 26 Nov 2020 pukul 16.47 Robert Forkel <
notifications@github.com> menulis:

**Pull request merged**